src/components/plugins/DummyPlugin2/DummyPlugin2.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
There is no need to create two separate files for the plugin we can use .map function for the same